src/components/common/helpers/Helpers.tsx

```diff
@@ -1194,7 +1194,10 @@ export const getDeploymentAppType = (
     return allowedDeploymentTypes[0]
 }
 
-export const hasApproverAccess = (email: string, approverList: string[]): boolean => {
+export const hasApproverAccess = (email: string = '', approverList: string[]= []): boolean => {// added null check for email and approverList
```

Comment:
Can remove equating to '' since already checking in first if block. And can remove this comment since it is clear from the if block.